ReactWindows/ReactNative.Shared/UIManager/BorderedCanvas.cs

```diff
@@ -40,5 +38,15 @@ public Border Border
                 Children.Insert(0, _border);
             }
         }
+
+        /// <inheritdoc />
+        protected override AutomationPeer OnCreateAutomationPeer()
+        {
+            return new DynamicAutomationPeer<BorderedCanvas>(this);
```

Comment:
If we are going to do this, the trick where we set AutomationProperties.Name to make Canvas create a peer always will not be needed.

Reply:
Same as for similar comment you made in ReactControl:

I would not optimize the trick in importantForAccessibility implementation out yet, because traits implementation is not dependent on your importantForAccessibility implementation, and there is a possibility, I hope, that it will be replaced by some for strict traits implementation like having component with explicit behavior for each trait.

---
In reply to: [168884083](https://github.com/gmangov/react-native-windows/pull/1#discussion_r168884083) [](ancestors = 168884083)